src/node_crypto.cc

```diff
@@ -741,6 +743,61 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UseExtraCaCerts(const char* fname) {
+  extra_root_certs_file = fname;
+}
+
+static void EmitWarning(Environment* env, const char* fmt, ...) {
+  char warning[1024];
+  va_list ap;
+
+  va_start(ap, fmt);
+  vsnprintf(warning, sizeof(warning), fmt, ap);
+  va_end(ap);
+
+  HandleScope handle_scope(env->isolate());
+  Context::Scope context_scope(env->context());
+
+  Local<Object> process = env->process_object();
+  Local<v8::Value> f = process->Get(
```

Comment:
Can you put the part after the '=' on the next line and indent by four spaces?  Also, can you use the overload that takes a `v8::Context` and returns a `v8::MaybeLocal<v8::Value>`?